10.9-libcxx/stable/main/finkinfo/sci/gmt5.info

```diff
@@ -31,7 +32,7 @@ CompileScript: <<
   cd build
   cmake \
 	-DCMAKE_BUILD_TYPE=Release \
-	-DCMAKE_C_FLAGS=-fstrict-aliasing \
+	-DCMAKE_C_FLAGS="-fstrict-aliasing -MD" \
```

Comment:
Why the addition of `-MD`?

Reply:
This is to allow using fink-package-precedence at the end of the compilation stage. This checks which Fink packages provided header files for the build.
Granted, it is a bit superfluous. Once tested, it can just as well be removed.

Reply:
Good idea to leave it there. It helps maintainers see what is being detected (BuildDepends/Depends) and helps debug when users get runtime problems (unexpected things detected). And it throws a fatal error if it detects use of /usr/local, which is otherwise a difficult-to-detect recipe for a Bad Experience.

Reply:
Thanks, Daniel, for the feedback on f-p-p. Looking forward to either of you approving the pull request.